### PR TITLE
Adding rcutorture test to avocado

### PR DESCRIPTION
**Pull request by PraveenPenguin**
Signed-off-by: praveen@linux.vnet.ibm.com
added rcutorture (new) Test case 

```
v1: check rcutorture module is in running kernel
v1: load rcutorture module
v1: on-line and off-line cpu operation 
v1: unload rcutorture module
v1:verify after run if there problem reported in dmesg 
v1: problem reported than notify user to evaluate the problem 
```

### CODE REVIEW

File: generic/rcutorture.py [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+        cmd = os.system('lsmod | grep rcutorture')
+        if cmd == 0:
+            self.log.info("module already loaded\n")
+            process.system('rmmod rcutorture')
+
+    def cpus_toggle(self):
+        """
+        Toggle CPUS online and offline
+        """
+        cmd = "lscpu | grep 'CPU(s):' |head -1| awk '{ print $2 }'"
```
Comment (apahim) [inline generic/rcutorture.py:56]: `multiprocessing.cpu_count()`

**Comment by apahim** [comment]
@PraveenPenguin, could you please provide a job.log of this test (using fpaste.org)?

**Comment by PraveenPenguin** [comment]
@apahim please find job.log in this link

https://paste.fedoraproject.org/396853/97136941/

Thanks

**Comment by PraveenPenguin** [comment]
@apahim  raised new PR #128 with changes , Please have a look 

Thanks

**Pull request closed without merging**